client/my-sites/importer/author-mapping-item.jsx

```diff
@@ -64,12 +60,16 @@ export default class extends React.PureComponent {
 				<Gridicon className="importer__mapping-relation" icon="arrow-right" />
 				{ ! hasSingleAuthor ? (
 					<AuthorSelector siteId={ siteId } onSelect={ onSelect }>
-						<UserItem user={ selectedAuthor } />
+						<User user={ selectedAuthor } />
 					</AuthorSelector>
 				) : (
-					<UserItem user={ this.getCurrentUser() } />
+					<User user={ this.getCurrentUser() } />
 				) }
 			</div>
 		);
 	}
 }
+
+export default connect( state => ( { currentUser: getCurrentUser( state ) } ) )(
```

Comment:
If you put the `currentUser` field on a separate line, Prettier will format this connect call a bit more nicely.

Reply:
good call